ui/package.json

```diff
@@ -23,10 +23,10 @@
     "@clr/core": "^3.0.0-next.8",
     "@clr/icons": "3.0.0-next.8",
     "@clr/ui": "3.0.0-next.8",
-    "@ngrx/effects": "^8.6.0",
-    "@ngrx/router-store": "^8.6.0",
-    "@ngrx/store": "^8.6.0",
-    "@ngrx/store-devtools": "^8.6.0",
+    "@ngrx/effects": "^9.0.0-beta.2",
```

Comment:
Why did you upgrade ngrx version to beta ?

Reply:
Because of this issue about Angular 9: https://github.com/ngrx/platform/issues/2362